src/main/java/org/killbill/billing/plugin/adyen/api/AdyenPaymentPluginApi.java

```diff
@@ -500,12 +502,7 @@ public HostedPaymentPageFormDescriptor buildFormDescriptor(final UUID kbAccountI
         }
 
         final String formUrl = hostedPaymentPagePort.getFormUrl(paymentData);
-
-        try {
-            return new AdyenHostedPaymentPageFormDescriptor(kbAccountId, formUrl, formParameter);
-        } catch (final URISyntaxException e) {
-            throw new PaymentPluginApiException("Unable to generate valid HPP url", e);
-        }
+        return new AdyenHostedPaymentPageFormDescriptor(kbAccountId, "GET", formUrl, PluginProperties.buildPluginProperties(formParameter), Collections.<PluginProperty>emptyList());
     }
```

Comment:
Created the PR: https://github.com/killbill/killbill-plugin-framework-java/pull/7